### PR TITLE
feat: v13 - Other/Miscallaneous changes.

### PR DESCRIPTION
**Pull request by WiseDevHelper**
**Please describe the changes this PR makes and why it should be merged:**

[Demo](https://deploy-preview-708--discordjs-guide.netlify.app/)

This PR updates some parts of the guide for the `v13` release. This PR is focused on updating things that are minor or too small to have a PR of their own.

**Status**

- [x] `message` -> `messageCreate`.
- [x] The `.reply()` method.
- [x] @everyone -> `@everyone`
- [x] MessageOptions.
- [x] Self-closing `<path>`

* Update the `FAQ` page. (Moved to #720)
* Update `Code Samples`. (Moved to #721)

Please let me know if there is more to change. I'll be happy to work on them.

### CODE REVIEW

File: guide/popular-topics/faq.md [hunk 1]
```diff
@@ -354,10 +368,11 @@ The second, **Roundtrip Latency**, describes the amount of time a full API round
 
 ### How do I play music from YouTube?
 
-For this to work, you need to have `ytdl-core` installed.
+For this to work, you need to have `ytdl-core` and `@discordjs/voice` installed.
 
 ```bash
 npm install --save ytdl-core
+npm install @discordjs/voice
```
Comment (almeidx) [inline guide/popular-topics/faq.md:375]:
```suggestion
npm install ytdl-core @discordjs/voice
```

**Review by almeidx — CHANGES_REQUESTED** [review]

File: guide/popular-topics/faq.md [hunk 2]
```diff
@@ -210,15 +211,15 @@ Mentions in embeds may resolve correctly in embed description and field values b
...
+new Discord.Client({ allowedMentions: { parse: ['users', 'roles'] } });
 ```
 
 Even more control can be achieved by listing specific `users` or `roles` to be mentioned by ID, e.g.:
 ```js
-message.channel.send('<@123456789012345678> <@987654321098765432>', {
-	allowedMentions: { users: ['123456789012345678'] },
+message.channel.send({
+	content: '<@123456789012345678> <@987654321098765432> <@&111111111111111111>',
+	allowedMentions: { users: ['123456789012345678'], roles: ['111111111111111111'] },
```
Comment (Danktuary) [inline guide/popular-topics/faq.md:222]:
```suggestion
	content: '<@123456789012345678> <@987654321098765432> <@&102938475665748392>',
	allowedMentions: { users: ['123456789012345678'], roles: ['102938475665748392'] },
```
No real reason; just to mix it up.

**Review by almostSouji — CHANGES_REQUESTED** [review]
Looking at the scoping of the tasks again, this should definitely not all be done in a single PR. Code Samples and FAQ at least should be handled in their own, separate PR's. Adding inline code and transforming the deprecated message to messageCreate are probably fine to do together, but otherwise this is too much of a scope mix to do all at once.

**Comment by WiseDevHelper** [comment]
@almostSouji Split into their own `PRs` 👍

File: guide/popular-topics/partials.md [hunk 3]
```diff
@@ -14,7 +14,7 @@ You choose which structures you want to emit as partials as client options when
 
 ```js
 const { Client } = require('discord.js');
-const client = new Client({ partials: ['MESSAGE', 'CHANNEL', 'REACTION'] });
+const client = new Client({ partials: ['messageCreate', 'CHANNEL', 'REACTION'] });
```
Comment (Danktuary) [inline guide/popular-topics/partials.md:17]:
```suggestion
const client = new Client({ partials: ['MESSAGE', 'CHANNEL', 'REACTION'] });
```

File: guide/popular-topics/reactions.md [hunk 4]
```diff
@@ -247,7 +247,7 @@ If you use [gateway intents](/popular-topics/intents.md) but can't or don't want
 
 ```js
 const Discord = require('discord.js');
-const client = new Discord.Client({ partials: ['MESSAGE', 'CHANNEL', 'REACTION'] });
+const client = new Discord.Client({ partials: ['messageCreate', 'CHANNEL', 'REACTION'] });
```
Comment (Danktuary) [inline guide/popular-topics/reactions.md:250]:
```suggestion
const client = new Discord.Client({ partials: ['MESSAGE', 'CHANNEL', 'REACTION'] });
```

**Comment by WiseDevHelper** [comment]
Rebased.

File: guide/additional-info/rest-api.md [hunk 5]
```diff
@@ -68,7 +68,7 @@ fetch('https://aws.random.cat/meow').then(response => response.json());
 It may seem like this does nothing, but what it's doing is launching a request to the random.cat server. The server is returning some JSON that contains a `file` property, which is a string containing a link to a random cat. node-fetch returns a response object, which we can change into JSON with `response.json()`. Next, let's implement this into a command. The code should look similar to this:
 
 ```js {3-6}
-client.on('message', async message => {
+client.on('messageCreate', async message => {
 	// ...
 	if (command === 'cat') {
 		const { file } = await fetch('https://aws.random.cat/meow').then(response => response.json());
```
Comment (Danktuary) [inline guide/additional-info/rest-api.md:74]: The `.send(file)` and `.send(embed)` lines in this file need to be updated to their respective `MessageOptions` properties

Comment (WiseDevHelper) [reply guide/additional-info/rest-api.md:74]: Resolved in #721.

**Review by Danktuary — APPROVED** [review]

**Pull request merged**